aiobotocore/endpoint.py

```diff
@@ -13,6 +14,8 @@
     logger,
 )
 from botocore.hooks import first_non_none_response
+from botocore.utils import lowercase_dict
+from moto.core.botocore_stubber import MockRawResponse
```

Comment:
moto is not a dependency of this library, so you can't do this.  I think this is really a bug in moto, it needs support for aiobotocore.  If you were to do this it would need to be a try/except ImportError.  Another option would be for aiobotocore to expose some moto helpers that correctly patch the response.  I'd do that instead as it will be a lot cleaner.

Reply:
> moto is not a dependency of this library, so you can't do this. 
Ack! I missed that! 
I'm a little busy this week but I'll try to sneak in some time to dig into moto repo and see if there's a cleaner solution to this. I tried sending encoded `raw_headers` [here](https://github.com/akshara08/moto/blob/bbb07b4e7ede832d69b8fe0089e0707c915710fc/moto/core/botocore_stubber.py#L69) the last time I tried and it broke a bunch of tests. Let me try that again and see

Reply:
no worries, thanks for working on this